app/io/pathfinder/controllers/ClusterController.java

```diff
@@ -0,0 +1,104 @@
+package io.pathfinder.controllers;
```

Comment:
Yea, I'm not suggesting private, but it's all in the same package so we could just use default